sandbox/pages/SandboxCodeBlock.vue

```diff
@@ -0,0 +1,219 @@
+          language="json"
+          searchable
+        />
+      </SandboxSectionComponent>
+      <SandboxSectionComponent
+        class="limited-width"
+        title="singleLine"
+      >
+        <KCodeBlock
+          id="single-line-prop"
+          :code="singleLineCode"
+          language="plaintext"
+          single-line
+        />
+      </SandboxSectionComponent>
+      <SandboxSectionComponent
+        class="limited-width"
+        title="processing"
+      >
+        <KCodeBlock
+          id="processing-prop"
+          :code="code"
+          language="json"
+          processing
+          searchable
+        />
+      </SandboxSectionComponent>
+      <SandboxSectionComponent
+        class="limited-width"
+        title="maxHeight"
+      >
+        <KCodeBlock
+          id="max-height-prop"
+          :code="code"
+          language="json"
+          max-height="200"
+          searchable
+        />
+      </SandboxSectionComponent>
+      <SandboxSectionComponent
+        class="limited-width"
+        title="query"
+      >
+        <KCodeBlock
+          id="query-prop"
+          :code="code"
+          language="json"
+          max-height="200"
+          query="compilerOptions"
+          searchable
+        />
+      </SandboxSectionComponent>
+      <SandboxSectionComponent
+        class="limited-width"
+        title="showCopyButton"
+      >
+        <KCodeBlock
+          id="show-copy-button-prop"
+          :code="code"
+          language="json"
+          searchable
+          :show-copy-button="false"
+        />
+      </SandboxSectionComponent>
+      <SandboxSectionComponent
+        class="limited-width"
+        title="showLineNumberLinks"
+      >
+        <KCodeBlock
+          id="show-line-number-links-prop"
+          :code="code"
+          language="json"
+          searchable
+          show-line-number-links
+        />
+      </SandboxSectionComponent>
+      <SandboxSectionComponent
+        class="limited-width"
+        title="theme"
+      >
+        <KCodeBlock
+          id="theme-prop"
+          :code="code"
+          initial-reg-exp-mode
+          language="json"
+          max-height="200"
+          processing
+          query="compilerOptions"
+          searchable
+          theme="dark"
+        />
+        <KCodeBlock
+          id="single-line-dark-prop"
+          :code="singleLineCode"
+          language="plaintext"
+          single-line
+          theme="dark"
+        />
+      </SandboxSectionComponent>
+
+      <!-- Slots -->
+      <SandboxTitleComponent
+        is-subtitle
+        title="Slots"
+      />
+      <SandboxSectionComponent
+        class="limited-width"
+        title="secondary-actions"
+      >
+        <KCodeBlock
+          id="secondary-actions-slot"
+          :code="code"
+          language="json"
+        >
+          <template #secondary-actions>
+            <KButton>
+              Secondary Action
+            </KButton>
```

Comment:
**Suggestion**: Should this use a KCodeBlockIconButton? That would help verify that using it has the right styles just by looking at the sandbox.